core/execution/market.go

```diff
@@ -3232,6 +3232,7 @@ func (m *Market) settlementPriceWithLock(ctx context.Context) {
 
 			// send the market data with all updated stuff
 			m.broker.Send(events.NewMarketDataEvent(ctx, m.GetMarketData()))
+			m.broker.Send(events.NewMarketSettled(ctx, m.GetID(), m.timeService.GetTimeNow().UnixNano(), m.markPrice, m.positionFactor))
 		} else {
```

Comment:
nit-pick: this `else` seems pointless. Return early, and remove the `else`

Reply:
Addressed with this commit https://github.com/vegaprotocol/vega/pull/6262/commits/dd3080400fa705880944db06480782ac0a45ee6b